apps/admin-gui/src/app/users/pages/user-detail-page/user-organizations/user-organizations.component.html

```diff
@@ -0,0 +1,39 @@
+<h1 *ngIf="this.showPrincipal"
```

Comment:
tbh this variable "showPrincipal" is not very describing for what it is needed. Rename it so it is more descriptive.

Reply:
same in users-groups